src/frontend/citizen-portal/src/app/components/disputant-form/disputant-form.component.html

```diff
@@ -4,7 +4,7 @@ <h4><b>Name on ticket</b></h4>
     <div class="row">
       <div class="col-md-6">
         <mat-form-field class="w-100">
-          <mat-label>Surname</mat-label>
+          <mat-label>Surname or Corporate Name</mat-label>
```

Comment:
To help address the absolutely huge tech dept for this project, whenever I edit hard-coded text, I use the better practice of the i18n files that Shark started. Move this text to the en.json and fr.json files and use this instead:
`{{ "label.surname_corporate_name" | translate }}`